### ISSUE TITLE
Implement fingerprint management for Digital Asset Links

### ISSUE THREAD

**Issue by andreban** [GoogleChromeLabs/bubblewrap#477]
### `bubblewrap fingerprint <subcommand>`:
Allows managing fingerprints in Bubblewrap. The list of fingerprints is saved on the fingerprints field in `twa-manifest.json`.
subcommands:

Options:

 - `--add <fingerprint>`
 - `--list`
 - `--remove <fingerprint>`

### `bubblewrap generate-assetlinks`
Generates the `assetlinks.json` file using the keys listed in `twa-manifest.json`. This list is bootstrapped when setting / generating the initial signing key.

**Comment by PEConn** [comment]
Is it worth fingerprints having a description? Something like

```
bubblewrap fingerprint --add 12:12:12:12:... --name debug
bubblewrap fingerprint --add 12:12:12:12:... --name "play generated"
```

This wouldn't change the output Digital Asset Links file, but would make them a bit easier to manage.

**Comment by andreban** [comment]
Started #478, with the `--name` flag.

**Comment by andreban** [comment]
Closed via #478
